src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -592,4 +623,14 @@ private ItemBuilder createSmallAngryPlanetItemBuilder() {
       .withVolume("smallAngryPlanetVolume")
       .withYearCaption(Collections.singletonList("2019"));
   }
+
+  private ItemBuilder createSmallAngryPlanetItemBuilder(UUID locationId, int barcode) {
```

Comment:
Do we need this to be a separate method, considering it's only called from one place?

Reply:
moved